### PR TITLE
Feature/modules

### PR DESCRIPTION
**Pull request by maosan132**
# Description

Hello pals, I am submitting this pull request so everybody can give support and suggestions for making this project better, so it can be approved. Any suggestions will be fully appreciated. 

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/contact.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      labore et dolore magna aliqua:</p>
+      <br>
+    <ul>
+      <li><strong>Contact Name: </strong> Bruce Wayne</li>
+      <li><strong>Phone number: </strong> +376456456</li>
+      <li><strong>Address: </strong> Street 123</li>
+      <li><strong>email: </strong> info@table.com</li>
+    </ul>
+  </div>
+  <img class="image" src= ${map} alt="">
```
Comment (bellom) [inline src/contact.js]: Let's make use of JS to generate the HTML structure.

This project requires us to use `document.createElement` and `document.appendChild` to appends HTML elements, just like you did in `line 4`.

we don't want the HTML structure to be hardcoded.

File: src/home.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+    labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+    aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore
+    eu
+    fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt
+    mollit
+    anim id est laborum.</p>
+  </div>
+  <img class="image" src= ${cover} alt="">
+  `;
```
Comment (bellom) [inline src/home.js]: Let's make use of JS to generate the HTML structure.

This project requires us to use `document.createElement` and `document.appendChild` to appends HTML elements, just like you did in `line 4`.

we don't want the HTML structure to be hardcoded.

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+const container = document.getElementById('content');
+
+const createMenuTabs = () => {
+  const tabs = document.createElement('div');
+  tabs.className = 'tabsContainer';
+  tabs.innerHTML = `
+    <button class="menulink" id="home">Home</button>
+    <button class="menulink" id="menu">Our Menu</button>
+    <button class="menulink" id="contact">Contact Us</button>
```
Comment (bellom) [inline src/index.js]: Let's make use of JS to generate the HTML structure.

This project requires us to use `document.createElement` and `document.appendChild` to appends HTML elements, just like you did in `line 8`.

we don't want the HTML structure to be hardcoded.

File: src/menu.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+      labore et dolore magna aliqua.</p>
+  </div>
+  <div class="module">
+    <img src=${item1} alt="">
+    <h2>Item #6</h2>
+    <span>Price: $200</span>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+      labore et dolore magna aliqua.</p>
+  </div>
```
Comment (bellom) [inline src/menu.js]: Let's make use of JS to generate the HTML structure.

This project requires us to use `document.createElement` and `document.appendChild` to appends HTML elements, just like you did in `line 6`.

we don't want the HTML structure to be hardcoded.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @maosan132  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: src/contact.js (hunk 1 above)
Comment (maosan132) [reply src/contact.js]: Thanks for the suggestions offered. I follow them and remove the HTML injected code, and use exclusively DOM traversing.

File: src/home.js (hunk 2 above)
Comment (maosan132) [reply src/home.js]: Thanks for the suggestions offered. I follow them and remove the HTML injected code, and use exclusively DOM traversing.

File: src/index.js (hunk 3 above)
Comment (maosan132) [reply src/index.js]: Thanks for the suggestions offered. I follow them and remove the HTML injected code, and use exclusively DOM traversing.

File: src/menu.js (hunk 4 above)
Comment (maosan132) [reply src/menu.js]: Thanks for the suggestions offered. I follow them and remove the HTML injected code, and use exclusively DOM traversing.

**Comment by Dandush03** [comment]
# Status: :rocket: Project Approved
### Great Job Applying all necessary changes
### Happy Coding :muscle:

**Pull request merged**